src/cpp/session/modules/SessionDataImportV2.R

```diff
@@ -13,6 +13,27 @@
 #
 #
 
+.rs.addFunction("path_relative_to_working_dir", function(target)
```

Comment:
nit: we typically use `snake_case` for RPC methods, and `camelCase` for regular `.rs` functions.